ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/QueryPlanningPromptTemplate.java

```diff
@@ -0,0 +1,111 @@
+package org.opensearch.ml.engine.tools;
+
+public class QueryPlanningPromptTemplate {
+
+    public static final String DEFAULT_QUERY =
+        "{ \"query\": { \"multi_match\" : { \"query\":    \"${parameters.query_text}\",  \"fields\": ${parameters.query_fields:-[\"*\"]} } } }";
+
```

Comment:
Why not just make it easy with text blocks? Is there a hard requirement to escape quotes. 
"""
    {
      "query": {
        "multi_match": {
          "query": "${parameters.query_text}",
          "fields": ${parameters.query_fields:-["*"]}
        }
      }
    }
    """